src/Microsoft.IdentityModel.Tokens/TokenUtilities.cs

```diff
@@ -136,6 +137,26 @@ internal static object GetClaimValueUsingValueType(Claim claim)
             return claim.Value;
         }
 
+        /// <summary>
+        /// Returns all <see cref="SecurityKey"/> provided in validationParameters and the configuration.
+        /// </summary>
+        /// <param name="validationParameters">A <see cref="TokenValidationParameters"/> required for validation.</param>
+        /// <returns>Returns all <see cref="SecurityKey"/> provided in validationParameters.</returns>
+        internal static IEnumerable<SecurityKey> GetAllSigningKeysWithConfig(TokenValidationParameters validationParameters)
+        {
+            LogHelper.LogInformation(TokenLogMessages.IDX10259);
+
+            StandardConfiguration configuration = null;
+            if (validationParameters.ConfigurationManager != null)
+                configuration = validationParameters.ConfigurationManager.GetStandardConfigurationAsync(CancellationToken.None).Result;
```

Comment:
Not sure if this Configuration should be retrieved in a single place and then passed on to other methods that need it.